Tools/autotest/models/Callisto.param

```diff
@@ -46,14 +47,22 @@ MOT_THST_HOVER,0.52
 MOT_YAW_HEADROOM,50
 PILOT_THR_BHV,3
 PILOT_Y_RATE,60
+PILOT_Y_RATE_TC,0.25
 PSC_ACCZ_FLTD,10
 PSC_ACCZ_FLTE,0
 PSC_ACCZ_FLTT,10
-PSC_ACCZ_I,1.0
+PSC_ACCZ_I,0.5
+PSC_ACCZ_IMAX,800
+PSC_ACCZ_P,0.25
 PSC_ANGLE_MAX,45
 PSC_JERK_Z,10
 PSC_POSZ_P,0.5
 PSC_VELZ_P,2.5
+RNGFND1_MAX_CM,10000
+RNGFND1_ORIENT,25
+RNGFND1_PIN,0
+RNGFND1_SCALING,12.1212

```

Comment:
I think some of these RNGFND1_ values are the defaults so we could remove them.  In particular I think these can be removed:

- RNGFND1_ORIENT,25
- RNGFND1_PIN,0
- RNGFND1_SCALING,12.1212

Reply:
Confirmed I need:
RNGFND1_PIN,0
RNGFND1_SCALING,12.1212
But I don't need: 
RNGFND1_ORIENT,25
![image](https://user-images.githubusercontent.com/100896/208243719-32e52441-de5e-48cf-85bf-12e4f2657123.png)

Reply:
RNGFND1_PIN defaults to -1 I think, so setting this to 0 would be better